### PR TITLE
[Improve]Modify code debugging steps and optimize image captions

### PR DESCRIPTION
**Pull request by cjj2010**
Modify code debugging steps and optimize image captions

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/user-guide/3-development.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ cd incubator-streampark/
 ./build.sh
 ```
 
-### 打开项目
+### 打开项目
```
Comment (caicancai) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/user-guide/3-development.md:26]: Why are there some extra spaces here?

**Review by caicancai — CHANGES_REQUESTED** [review]

**Comment by caicancai** [comment]
Thank you for your contribution and I look forward to your next contribution

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/user-guide/3-development.md [hunk 2]
```diff
@@ -74,11 +74,14 @@ pnpm serve
 ![Streampark Select Team](/doc/image/streampark_select_team.jpg)
 
 ### 演示调试代码
+1.在idea以debug模式启动项目
 
-选择`Project`菜单，点击build
+2.在ApplicationController的flink/app/list加入断点
```
Comment (caicancai) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/user-guide/3-development.md:79]: cc@wolfboys I'm not sure if this modification is appropriate.

**Comment by caicancai** [comment]
cc @wolfboys

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/user-guide/3-development.md (hunk 1 above)
Comment (cjj2010) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/user-guide/3-development.md]: fix now

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**